client/src/stories/Button.js

```diff
@@ -1,50 +1,32 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import './button.css';
-
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+import ButtonPlus from "./assets/buttonPlus.svg";
 /**
  * Primary UI component for user interaction
  */
-export const Button = ({ primary, backgroundColor, size, label, ...props }) => {
-  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
+
+const ButtonElement = styled.div`
```

Comment:
You can be less (and more) specific here.
If this `div` is only used inside this button, you don't have to add `Button` prefix. 
On the other side, `Element` is very generic.

```suggestion
const Container = styled.div`
```

```suggestion
const Wrapper = styled.div`
```